### PR TITLE
hoc2015: update callouts, and a few level tweaks.

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

**Comment by breville** [comment]
fyi @cpirich

File: apps/i18n/studio/en_us.json [hunk 1]
```diff
@@ -22,8 +22,16 @@
...
+  "calloutClickEvents": "Click on the events header to see event function blocks.",
   "calloutTryOutNewCommands": "Try out all the new commands you’ve unlocked",
-  "calloutUseArrowButtons": "Hold down these buttons or the arrow keys on your keyboard to trigger the move events.",
+  "calloutUseArrowButtons": "Hold down these buttons or the arrow keys on your keyboard to trigger the move events",
+  "calloutRunButton": "Add a moveRight command to your code and then click here to run it",
+  "calloutShowCodeToggle": "Click here to switch between block and text mode",
+  "calloutShowPlaceGoUpHere": "Place goUp command here to move up",
+  "calloutShowPlaySound": "It's your game, so you choose the sounds now. Try the dropdown to pick a different sound",
+  "calloutInstructions": "Don't know what to do? Click the instructions to see them again",
+  "calloutPlaceTwo": "Can you make two MOUSEs appear when you get a BIRD?",
```
Comment (islemaster) [inline apps/i18n/studio/en_us.json:33]: "MICE"?

File: apps/src/studio/levels.js [hunk 2]
```diff
@@ -1963,21 +1986,21 @@ levels.js_hoc2015_event_two_items = {
   'callouts': [
     {
       'id': 'playlab:js_hoc2015_event_two_items:placeCommandsHere',
-      'element_id': '.droplet-main-canvas',
+      'element_id': '.droplet-gutter-line:nth-of-type(2)',
```
Comment (islemaster) [inline apps/src/studio/levels.js:1989]: :+1:

**Comment by islemaster** [comment]
LGTM

**Pull request merged**